samples/bot-sharepoint-file-viewer/csharp/BotWithSharePointFileViewer/AppManifest/manifest.json

```diff
@@ -0,0 +1,62 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.11",
+  "version": "1.0.0",
+  "id": "<<Microsoft-App-Id>>",
+  "packageName": "com.microsoft.teams.botsharepointfileviewer",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.microsoft.com/privacy",
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+  },
+  "name": {
+    "short": "Bot SharePoint file viewer",
+    "full": "Bot SharePoint file viewer"
+  },
+  "description": {
+    "short": "Bot SharePoint file viewer",
+    "full": "This sample demos a bot which uploads file to SharePoint site and same files can be viewed in Teams file viewer"
```

Comment:
This sample demos a bot with capability to upload files to SharePoint site and same files can be viewed in Teams file viewer.